pkg/csi/cinder/openstack/openstack.go

```diff
@@ -169,21 +162,15 @@ func CreateOpenStackProvider() (IOpenStack, error) {
 		klog.Errorf("GetConfigFromFiles %s failed with error: %v", configFiles, err)
 		return nil, err
 	}
-	if strings.ToLower(os.Getenv("USE_CLAYMORE")) == "true" {
-		updateConfigFromVars(&cfg)
-	}
 	logcfg(cfg)
-
 	provider, err := client.NewOpenStackClient(&cfg.Global, "cinder-csi-plugin", userAgentData...)
 	if err != nil {
 		return nil, err
 	}
-
 	epOpts := gophercloud.EndpointOpts{
 		Region:       cfg.Global.Region,
 		Availability: cfg.Global.EndpointType,
 	}
-
```

Comment:
Nit (and personal opinion):  These newlines in between blocks of code help improve readability in general.